src/components/ListPage.js

```diff
@@ -8,10 +8,14 @@ const ListPage = props => {
     <div className="row add-pets-section">
       <div className="small-12 medium-6 columns about-pets">
         <div className="about-pets-avatar">
+        <Link to={`/pets/${id}`}>
           <img className="avatar-image" src={img_url} alt={name} />
+          </Link>
```

Comment:
This link should also be backed up.

Reply:
So you want the <img> and <Link> to be spaced the same or that you want <img> to be more indented?

Reply:
OK updated https://github.com/xshigeox/adopt_a_pet.git